aquality-automation/aquality-tracking-ui
# Feature/test view suite link leads to 404 page

# PR Details

Links to suite on test view page leaded to 404 page instead of suite view

## Related Issue

#39 

## How Has This Been Tested

1. Open Test View Page
1. Click Suite link
1. Should redirect to Suite View Page

## Checklist
